doc/1/getting-started/standalone/index.md

```diff
@@ -0,0 +1,154 @@
+---
+code: false
+type: page
+title: C# Standalone
+description: Getting started with Kuzzle and C#
+order: 0
+---
+
+# Getting Started with Kuzzle and C#
+
+This tutorial explains you how to use **Kuzzle** with **C#**, **.NET Core SDK 2.1** and the **Kuzzle C# SDK**.
+It will walk you through creating scripts that can **store** documents in Kuzzle and subscribe to **notifications** about document creations.
+
+You are going to write an application that **stores** documents in Kuzzle Server and subscribe to **real time notifications** for each created document.
+
+To follow this tutorial, you must have a Kuzzle Server up and running. Follow these instructions if this is not already the case: [Running Kuzzle](/core/1/guides/getting-started/running-kuzzle/).
```

Comment:
```suggestion
To follow this tutorial, you must have a Kuzzle Server up and running. Follow these steps if it's not done yet: [Running Kuzzle](/core/1/guides/getting-started/running-kuzzle/).
```

Reply:
I find the original version better than this proposal.